msal/src/Microsoft.Identity.Client/Features/DeviceCode/IPublicClientApplication.cs

```diff
@@ -0,0 +1,72 @@
+﻿//------------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Interface to be used with desktop or mobile applications (Desktop / UWP / Xamarin.iOS / Xamarin.Android).
+    /// public client applications are not trusted to safely keep application secrets, and therefore they only access Web APIs in the name of the user only 
+    /// (they only support public client flows). For details see https://aka.ms/msal-net-client-applications
+    /// </summary>
+    public partial interface IPublicClientApplication : IClientApplicationBase
+    {
+        /// <summary>
+        /// Acquires device code from the authority and returns it to the caller via
+        /// the deviceCodeResultCallback.  The function then proceeds to poll for the security

```

Comment:
>  [](start = 42, length = 1)

nit: extra space

Reply:
Do you mean extra space after the period?

https://www.quickanddirtytips.com/education/grammar/two-spaces-after-a-period

yes, i'm one of those who was taught to put 2 spaces after a period and it's been cemented into my muscle memory.  I'll fix these.

Reply:
thanks for educating me. fun to learn something new :)

---
In reply to: [220763901](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1275#discussion_r220763901) [](ancestors = 220763901)